create_config

```diff
@@ -1,24 +1,23 @@
-#!/bin/zsh
-
-i3_config_file="$HOME/.i3/config"
-i3_includes_file="$HOME/.i3/config.d/i3_configs.conf"
+#!/bin/bash
+i3_folder="$HOME/.i3"
+i3_config_file="$i3_folder/config"
+i3_config_dir="$i3_folder/config.d"
+i3_includes_file="$i3_config_dir/i3_configs.conf"
 
 if [[ -f "$i3_includes_file" ]]; then
   [[ -f "$i3_config_file" ]] && rm "$i3_config_file"
 
   touch "$i3_config_file"
 
   echo "#### DO NOT EDIT THIS FILE, IT IS GENERATED." >> "$i3_config_file"
-  echo "#### EDIT THE FILES HERE: ~/.i3/config.d/**/*" >> "$i3_config_file"
+  echo "#### EDIT THE FILES HERE: $i3_folder/config.d/**/*" >> "$i3_config_file"
 
   include_file() {
-    cat "$HOME/.i3/config.d/$1" >> "$i3_config_file"
+    cat "$i3_config_dir/$1" >> "$i3_config_file"
   }
 
   get_file_name_from_line () {
-    removed_semicolon="${1%;*}"
-    removed_import="${removed_semicolon#import}"
-    stripped_file_name="$(echo $removed_import | sed 's/ //')"
+    stripped_file_name=$(echo $* | sed 's/\(import \)\(.*.conf\)\(;\)/\2/')
```

Comment:
much cleaner 🎉

Reply:
Hey thanks for that! And jump back, i3 is life haha, I mixed your idea with KDE Plasma 5, and it's neat! Put the script on xsessions just had to modify to get the `.sh` on it, xsessions didn't like otherwise